pandas/_libs/tslibs/conversion.pyi

```diff
@@ -12,30 +12,16 @@ class OutOfBoundsTimedelta(ValueError): ...
 
 def precision_from_unit(
     unit: str,
-) -> tuple[
-    int,  # int64_t
-    int,
-]: ...
-
-
+) -> tuple[int, int,]: ...  # (int64_t, _)
```

Comment:
havent seen this pattern elsewhere?

Reply:
can we use this pattern `# (comment for A,  comment for B)` for the return type of `tuple[A, B]`? It's more explicit than the one reformated by `black`